pkg/cmd/dev/BUILD.bazel

```diff
@@ -26,6 +26,7 @@ go_library(
         "//pkg/cmd/dev/io/exec",
         "//pkg/cmd/dev/io/os",
         "@com_github_alessio_shellescape//:shellescape",
+        "@com_github_cockroachdb_errors//:errors",
```

Comment:
`errors` is a pretty heavy dependency, can we please continue using the stdlib `errors`? I've been deliberately avoiding `cockroachdb/errors` for `dev`. 

My specific worry here is that `errors` depends on protobuf files so building the library requires building `protoc` which requires a working C++ toolchain, which you might not have by the time you run `dev doctor`, and in fact `dev doctor` tries to guide you toward getting a working C++ toolchain (via installing XCode/using a cross config/etc.) So people can find themselves in a situation where they cannot build `dev` to run `dev doctor` which is the thing that would tell them how to be able to build `dev`.